### ISSUE TITLE
Build crashes for directories with only images

### ISSUE THREAD

**Issue by fson** [gatsbyjs/gatsby#39]
If there is a folder in `pages` that includes only images (no jsx/md/html pages), the build crashes with an error `Cannot read property 'path' of undefined`:
```
> mkdir pages/images
> touch pages/images/foo.jpg
> gatsby build
Generating static html pages
Compiling production bundle.js
Copying assets
/Users/ville/Projects/gatsby/lib/utils/post-build.coffee:37
        newPath = parsePath(page.path).dirname + parsed.basename;
                                ^

TypeError: Cannot read property 'path' of undefined
  at copy (/Users/ville/Projects/gatsby/lib/utils/post-build.coffee:31:16)
  at /Users/ville/Projects/gatsby/node_modules/async/lib/async.js:358:13
  at /Users/ville/Projects/gatsby/node_modules/async/lib/async.js:239:13
```

**Comment by fson** [comment]
Thanks @KyleAMathews!

**Comment by KyleAMathews** [comment]
NP :) Published a 0.4.1 release to NPM.
